tests/core/data/vecfield/test_vecfield_fixtures.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef PHARE_TEST_CORE_DATA_TEST_VECFIELD_FIXTURES_HPP
+#define PHARE_TEST_CORE_DATA_TEST_VECFIELD_FIXTURES_HPP
+
+#include "tests/core/data/tensorfield/test_tensorfield_fixtures.hpp"
+
+namespace PHARE::core
+{
+template<std::size_t dim>
```

Comment:
could be a using statement instead maybe

[looks like](https://github.com/PhilipDeegan/PHARE/blob/mkn/tests/core/data/vecfield/test_vecfield_fixtures.hpp#L13)